xml/System.CodeDom.Compiler/CodeDomProvider.xml

```diff
@@ -156,10 +156,14 @@
   
 > [!NOTE]
 >  In the .NET Framework versions 1.0 and 1.1, this method is provided by the <xref:System.CodeDom.Compiler.ICodeCompiler> implementation that is returned by the <xref:System.CodeDom.Compiler.CodeDomProvider.CreateCompiler%2A> method of the provider. In version 2.0, this method can be called directly on the code provider even if it is not overridden by the code provider. If the code provider does not override this method, the <xref:System.CodeDom.Compiler.ICodeCompiler> implementation is called by the base class.  
+
+> [!NOTE]
+> On .NET Core and NET 5+, calls to the `CodeDomProvider.CompileAssemblyFromDom` method throw a <xref:System.PlatformNotSupportedException>; compile code is not supported.
```

Comment:
@gewarren  what is the preferred term for "the .NET Core lineage "

Reply:
If this is correct (this is what I've been using), there's still a minor correction that would be needed (in all places):

```suggestion
> On .NET Core and .NET 5+, calls to the `CodeDomProvider.CompileAssemblyFromDom` method throw a <xref:System.PlatformNotSupportedException>; compile code is not supported.
```

Reply:
I think that verbiage is fine. We've also been writing it as ".NET 5+ (including .NET Core versions)". But I think either is fine.